client/core/core.go

```diff
@@ -1726,6 +1729,16 @@ func (c *Core) CreateWallet(appPW, walletPW []byte, form *WalletForm) error {
 		// Balance and Address are set after connect.
 	}
 
+	if walletInfo.IsInternalWallet {
+		encXPrivKey, xPubKey, err := c.deriveWalletSeed(assetID, appPW)
+		if err != nil {
+			return fmt.Errorf("unable to derive wallet seed: %w", err)
```

Comment:
Seems very reasonable to me, but you'll want to coordinate with @buck54321 on this approach since he has done the same thing on his btc spv branch, in his own way.  It's very good to get two approaches and agree on the best.

Reply:
Ah I'll check it out.

Reply:
https://github.com/buck54321/dcrdex/commit/2165bc06754a40cf66d8e485995e3d60e5168d41#diff-13a4a6b12c1ae844ef90151638f194427cfdc545fceb58d52c0f6f33b81d859bR1637, but this is months behind master now and he probably has a different idea about it given how hd keys have settled and other issues.

Reply:
@martonp has it right.